.github/workflows/run-unit-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: Run unit tests
+
+on:
+  pull_request:
+    types: [opened,reopened,synchronize]
+
+jobs:
+  test-unit-jest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+
+      - run: corepack enable
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+          cache: yarn
```

Comment:
why do we need two steps to set up Node?

Reply:
I have no idea, it has probably something to do with caching. I copied this from the other workflows, where it is done the same way. If I leave out one of the "Setup Node.js" steps, then it crashes. I tried to make it only setup once, but no success so far.

Reply:
@davidmurdoch https://github.com/actions/setup-node/issues/480 I found why it is needed. Basically because of corepack.

Reply:
Fixed, I had to put corepack enable before installing node (this makes no sense lol)